elm.json

```diff
@@ -51,4 +52,4 @@
     "test-dependencies": {
         "elm-explorations/test": "1.2.2 <= v < 2.0.0"
     }
-}
+}
```

Comment:
Re-add the new line at EOF.
(Your editor should be respecting our `.editorconfig`)